services/core/src/utils/types.ts

```diff
@@ -17,6 +17,8 @@ export interface InjectorInput {
   contract: CheckedContract;
   bytecode?: string;
   creationData?: string;
+  constructorArguments?: string;
+  msgSender?: string;
```

Comment:
isn't it better to handle the the global variables as an object in which msgSender is one of the properties?